djangoBookStoreApi/views/user_profile_view.py

```diff
@@ -10,3 +12,61 @@ def get(self, request, user_id):
         user = get_object_or_404(Users, id=user_id)
         data = serializers.serialize('json', [user])
         return JsonResponse(data, safe=False)
+
+
+class CreateUserView(View):
+    def post(self, request):
+        # Extract JSON data from the request body
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = {}
+
+        # Extract username from the JSON data
+        username = data.get('username')
+
+        # Check if username is provided and not empty
+        if username:
```

Comment:
we need to make checks for if they have username and password at the beginning of this function